metering/metering-uninstall.adoc

```diff
@@ -1,13 +1,27 @@
 :context: metering-uninstall
 [id="metering-uninstall"]
-= Uninstalling Metering
+= Uninstalling metering
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
 You can remove metering from your {product-title} cluster.
 
-include::modules/metering-uninstall.adoc[leveloffset=+1]
+[NOTE]
+====
+Metering does not manage or delete Amazon S3 bucket data. You must manually clean up S3 buckets used to store metering data.
```

Comment:
To answer this - no, we wouldn't normally say anything about an open BZ that might fix this soon. Just whenever the BZ is fixed, then we will come into the docs and will remove this note, if that's appropriate. So just be sure that this stays on the radar to update docs after the BZ is fixed.

Reply:
Also, suggest specifying here that you must manually clean them up after uninstalling. Something like this, if it's accurate:

```
After uninstalling metering, you must manually clean up S3 buckets that were used to store metering data.
```

Reply:
> Also, suggest specifying here that you must manually clean them up after uninstalling. Something like this, if it's accurate:
> 
> ```
> After uninstalling metering, you must manually clean up S3 buckets that were used to store metering data.
> ```

@timflannagan1, is Andrea's suggestion more accurate to use instead of the original wording for the note?

Reply:
@lbarbeevargas There's no mechanism in metering that allows us to delete the actual s3 bucket and it's contents, at least in its current iteration, so I think Andrea's suggestion is fine with me.